tools/docker/linux_headers_image/Dockerfile

```diff
@@ -14,12 +14,13 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu:18.04@sha256:8aa9c2798215f99544d1ce7439ea9c3a6dfd82de607da1cec3a8a2fae005931b
+# debhelper-compat (= 12) needed for kernels >= 6.3
+FROM ubuntu:20.04@sha256:6d8d9799fe6ab3221965efac00b4c34a2bcc102c086a58dff9e19a08b913c7ef
 
 # Install required packages
 RUN apt-get update
 RUN apt-get upgrade -y -q
-RUN apt-get install -y -q build-essential \
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -q build-essential \
```

Comment:
nit: just add a `ENV DEBIAN_FRONTEND noninteractive` near the top instead so that it applies to the rest of the commands.

Reply:
Updated in 50103d2e7ff4df5aacd8252016660c6e699e61f0